src/core/curves.js

```diff
@@ -98,16 +98,30 @@ p5.prototype.bezier = function() {
  * @param {Number} detail resolution of the curves
  * @chainable
  * @example
- * <div>
+ * <div modernizr='webgl'>
  * <code>
- * background(204);
- * bezierDetail(50);
- * bezier(85, 20, 10, 10, 90, 90, 15, 80);
+ * function setup() {
+ *   createCanvas(100, 100, WEBGL);
+ *   noFill();
+ * }
+ *
+ * function draw() {
+ *   background(200);
+ *
+ *   // a triangle-wave function
+ *   bezierDetail(2 + abs(15 - floor(millis() / 150) % 30));
```

Comment:
this is very complex. can you change to the most simple case. maybe just setting the bezierDetail to a single number?